server/resources/ats.json

```diff
@@ -6,9 +6,8 @@
         "defaultConfigurationInstructionsHTML": "Configure JAWS with default settings. For help, read &lt;a href=&quot;https://github.com/w3c/aria-at/wiki/Configuring-Screen-Readers-for-Testing&quot;&gt;Configuring Screen Readers for Testing&lt;/a&gt;.",
```

Comment:
Looks like you're accidentally removing newly added tokens coming from w3c/aria-at. You may want to remove these changes.